### DISCUSSION TITLE
How do I delete a user from the auth.users table

### DISCUSSION THREAD

**Discussion by xt42io** [supabase/supabase#9621] [Questions] [answered]
I'm trying to delete a user from the users table.
I ran this command I found on [https://github.com/supabase/supabase/discussions/250](https://github.com/supabase/supabase/discussions/250)
`delete from auth.users
where id = THE ID OF THE USER TO BE DELETED`

It didn't work, and I got this error
`Failed to validate sql query: syntax error at or near "bb396"`

How do I actually delete a user?

**Comment by GaryAustin1** [comment, marked as answer]
Your error is likely because you did not put the uuid in single quotes 'uuid string'.  You may also need to add ::uuid to the end of the string.

**Reply by xt42io** [reply]
Wow, thanks. It worked!
